<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SDCentwidget {
    border-radius: 10px;
    background: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 #A7C5BD,
        stop:1 #6F9EAF
    );
    color: #2E3A59;  /* 字体颜色保持深色，清晰 */
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="SDCentwidget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="headWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>个性设置</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>593</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="CMWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="miniBut">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#miniBut{
background-color:transparent;
border:none;
}
#miniBut:pressed{
background-color:rgb(238, 238, 236);
border:none;
}
#miniBut:hover{
background-color:rgb(239, 41, 41);
border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="qic.qrc">
                <normaloff>:/qic/svg/mini.svg</normaloff>:/qic/svg/mini.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="closeBut">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#closeBut{
background-color:transparent;
border:none;
}
#closeBut:pressed{
background-color:rgb(238, 238, 236);
border:none;
}
#closeBut:hover{
background-color:rgb(239, 41, 41);
border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="qic.qrc">
                <normaloff>:/qic/images/close.png</normaloff>:/qic/images/close.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="setsWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="speedWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="speedLabel">
              <property name="text">
               <string>默认倍数:        </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="speedComb">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <item>
               <property name="text">
                <string>1x</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>0.75x</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1.5x</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2x</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="ThemeWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="ThemeLabel">
              <property name="text">
               <string>默认主题:         </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="ThemeComb">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <item>
               <property name="text">
                <string>暗夜主题</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>亮色主题</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>自定义颜色</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>跟随情绪</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>逆转情绪</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>智能感知推荐</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="playModeWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="playModeLabel">
              <property name="text">
               <string>默认播放模式:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="playModeComb">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <item>
               <property name="text">
                <string>顺序播放</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>随机播放</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>单个播放</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>单个循环</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qic.qrc"/>
 </resources>
 <connections/>
</ui>
